ui/js/controllers/perf/graphs.js

```diff
@@ -894,30 +894,37 @@ perf.controller('TestChooserCtrl', function($scope, $modalInstance, $http,
             $scope.timeRange).then(function(platformList) {
                 $scope.platformList = platformList.platformList;
                 $scope.platformList.sort();
-                $scope.selectedPlatform = $scope.platformList[0];
+                if (_.contains($scope.platformList, defaultPlatform)) {
+                    $scope.selectedPlatform = defaultPlatform;
+                } else {
+                    $scope.selectedPlatform = $scope.platformList[0];
```

Comment:
I don't think it's needed. We already preserve the preference if the user switches branches (see updateTestSelector below)